cg/meta/backup/backup.py

```diff
@@ -49,18 +50,20 @@ def __init__(
 
     def check_processing(self) -> bool:
         """Check if the processing queue for flow cells is not full."""
-        processing_flow_cells = self.status.flowcells(status=FlowCellStatus.PROCESSING).count()
-        LOG.debug(f"Processing flow cells: {processing_flow_cells}")
-        return processing_flow_cells < MAX_PROCESSING_FLOW_CELLS
+        processing_flow_cells_count: int = self.status.get_flow_cells_by_statuses(
+            flow_cell_statuses=[FlowCellStatus.PROCESSING]
+        ).count()
+        LOG.debug(f"Processing flow cells: {processing_flow_cells_count}")
+        return processing_flow_cells_count < MAX_PROCESSING_FLOW_CELLS
 
-    def get_first_flow_cell(self) -> Optional[models.Flowcell]:
+    def get_first_flow_cell(self) -> Optional[Flowcell]:
         """Get the first flow cell from the requested queue."""
-        flow_cell: Optional[models.Flowcell] = self.status.flowcells(
-            status=FlowCellStatus.REQUESTED
+        flow_cell: Optional[Flowcell] = self.status.get_flow_cells_by_statuses(
+            flow_cell_statuses=[FlowCellStatus.REQUESTED]
```

Comment:
There are two writing variants for flowcell - Flowcell and FlowCell. In function calls flow_cell gets used. 
Maybe we want to change all Flowcell to FlowCell for consistency.

Reply:
Agreed - but out of scope for this PR as it would require changes to the  DB. One of the future final refactor PR I think